proxy.py

```
@@ -1108,6 +1120,48 @@ async def scaffolding_apply(body: ScaffoldRequest, auth: AuthContext = Depends(v
+            )
+        if body.force:
+            added = await sr.refresh_remote_force()
+        else:
+            added = await sr.refresh_remote()
+        return {
+            "ok": True,
+            "added": added,
+            "total": len(sr.list()),
+            "forced": body.force,
+        }
+    except Exception as exc:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)